src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -391,9 +391,13 @@ String generateDeclarations(
       List<SourceFile> sourceFiles, List<SourceFile> externs, Depgraph depgraph)
       throws AssertionError {
     compiler.compile(externs, sourceFiles, opts.getCompilerOptions());
-    precomputeChildLists();
-    collectTypedefs();
-    String dts = produceDts(depgraph);
+    String dts = "";
```

Comment:
Currently, it seems like we always write the file - errors or no errors. Does blaze rely on that behavior. I am fearful of breaking something if we don't write anything in presence of errors.